docs/pages/render-hook.page.server.mdx

```diff
@@ -1,3 +1,13 @@
+import { Link, Warning } from '@brillout/docpress'
+
+<Warning>
+  `render()` is deprecated in favor of `onRenderHtml()`, see
+  <ul>
+    <li><Link href="/onRenderHtml" /></li>
+    <li><Link href="/migration/v1-design#renamed-hooks" /></li>
+  </ul>
+</Warning>
```

Comment:
I think we can also be lazy and just remove that page - as you wish.

Reply:
Good to know :pray: So far it's easy to keep these old pages but if it becomes cumbersome I'll remove them